dist/.magento.env.yaml

```diff
@@ -205,6 +205,20 @@
 #            deploy:                                                                                                  #
 #              MYSQL_USE_SLAVE_CONNECTION: true                                                                       #
 #######################################################################################################################
+# SPLIT_DB - use this variable to run a split database process during the deployment phase.                           #
+#            The split database process is irreversible.                                                              #
+#            Deleting a value will not produce the opposite result. You can only add a new type for splitting        #
+# Magento Version: 2.1.4 and later                                                                                    #
+# Default value: not set                                                                                              #
+# Possible values: 'sales' or/and 'quote'                                                                              #
+# Stages: deploy                                                                                                      #
+# Example:                                                                                                            #
```

Comment:
this file will be removed, please move example into `config/schema.yaml`

Reply:
done